Python/Libraries/Top_10_Python_Libraries_Video.md

```diff
@@ -0,0 +1,3 @@
+## Top_10_Python_Libraries_Video
+
```

Comment:
Just mention the names of the libraries that you've explained in the video

Reply:
In Description? @apeksha235

Reply:
@apeksha235 
Done! Now is it ok?

Reply:
@Anujakumari Also mention that you've explained these in the video

Reply:
@apeksha235 , Now is it ok?

Reply:
@apeksha235 , plz check once.